hyp3_gamma/rtc/rtc_sentinel.py

```diff
@@ -425,6 +451,8 @@ def main():
                         help='Radiometry of the output backscatter image(s)')
     parser.add_argument('--scale', choices=('power', 'amplitude'), default='power',
                         help='Scale of the output backscatter image(s)')
+    parser.add_argument('--power-unit', choices=('power', 'db'), default='power',
```

Comment:
Instead of adding another CLA (*--power-unit*), it would probably be better to use the existing variable *scale*.  Add dB as a 3rd option, i.e. power, amp, and dB. This is generally how we think of them - as three separate scales.